selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/dashboard/CreateWorkspaceTest.java

```diff
@@ -133,13 +133,13 @@ public void checkFiltersStacksFeature() {
     newWorkspace.clickOnMultiMachineTab();
     assertFalse(newWorkspace.isStackVisible(JAVA.getId()));
 
-    // filter stacks by 'php' value and check filtered stacks list
+    // filter stacks by 'blank' value and check filtered stacks list
```

Comment:
please, leave comment about why we need to user `blank` stack name

Reply:
We used here blank stack because **PHP** stack not existed on **OSIO**.